### PR TITLE
parallel universe

### PR DESCRIPTION
**Pull request by smriad**

### CODE REVIEW

**Review by ayushrana182 — COMMENTED** [review]
Shouldn't check markdown links pass the tests? Maybe check the links?

**Review by jbampton — COMMENTED** [review]
I am looking into the Check markdown links problem.

Might be that it's looking for `master` branch and we renamed it to `main`

**Comment by jbampton** [comment]
Screen Shot 2022-04-23 at 5 17 19 pm

**Comment by jbampton** [comment]
https://github.com/gaurav-nelson/github-action-markdown-link-check

Screen Shot 2022-04-23 at 5 19 19 pm

**Review by jbampton — CHANGES_REQUESTED** [review]
This needs to be removed completely not just commented out

**Comment by smriad** [comment]
> This needs to be removed completely not just commented out

Okay

**Comment by smriad** [comment]
> This needs to be removed completely not just commented out

@jbampton Please check now

**Review by jbampton — CHANGES_REQUESTED** [review]
File permissions are wrong.  Also `pages/parallel.html` should be removed.

File: docs/_config.yml [hunk 1]
```diff
@@ -67,6 +67,7 @@ exclude:
   - .gitignore
   - assets/python/
   - server
+  - pages/parallel.html
```
Comment (jbampton) [inline docs/_config.yml:70]:
```suggestion
```

**Review by jbampton — APPROVED** [review]

**Pull request merged**